src/main/java/edu/brown/library/repository/OcflHttp.java

```diff
@@ -448,30 +520,30 @@ public void handle(String target,
             handleRoot(response);
         }
         else {
-            var matcher = ObjectIdFilesPattern.matcher(updatedRequestURI);
-            if(matcher.matches()) {
-                var objectId = matcher.group(1);
-                handleObjectFiles(request, response, objectId);
-            }
-            else {
-                var matcher2 = ObjectIdPathContentPattern.matcher(updatedRequestURI);
-                if (matcher2.matches()) {
-                    var objectId = URLDecoder.decode(matcher2.group(1), StandardCharsets.UTF_8.toString());
-                    var path = URLDecoder.decode(matcher2.group(2), StandardCharsets.UTF_8.toString());
-                    handleObjectPathContent(request, response, objectId, path);
-                }
-                else {
-                    var matcher3 = ObjectIdPathPattern.matcher(updatedRequestURI);
-                    if (matcher3.matches()) {
-                        var objectId = URLDecoder.decode(matcher3.group(1), StandardCharsets.UTF_8.toString());
-                        var path = URLDecoder.decode(matcher3.group(2), StandardCharsets.UTF_8.toString());
-                        handleObjectPath(request, response, objectId, path);
-                    }
-                    else {
-                        response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+            try {
+                var matcher = ObjectIdFilesPattern.matcher(updatedRequestURI);
+                if (matcher.matches()) {
+                    var objectId = matcher.group(1);
+                    handleObjectFiles(request, response, objectId);
+                } else {
+                    var matcher2 = ObjectIdPathContentPattern.matcher(updatedRequestURI);
+                    if (matcher2.matches()) {
+                        var objectId = URLDecoder.decode(matcher2.group(1), StandardCharsets.UTF_8.toString());
+                        var path = URLDecoder.decode(matcher2.group(2), StandardCharsets.UTF_8.toString());
+                        handleObjectPathContent(request, response, objectId, path);
+                    } else {
+                        var matcher3 = ObjectIdPathPattern.matcher(updatedRequestURI);
+                        if (matcher3.matches()) {
+                            var objectId = URLDecoder.decode(matcher3.group(1), StandardCharsets.UTF_8.toString());
+                            var path = URLDecoder.decode(matcher3.group(2), StandardCharsets.UTF_8.toString());
+                            handleObjectPath(request, response, objectId, path);
+                        } else {
+                            response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+                        }
                     }
                 }
             }
+            catch(URISyntaxException e) {}
```

Comment:
Should it do something if there is a URISyntaxException?  Log or let the user know there is an error?

Or is nothing the desired behavior?